docs/introduction.rst

```diff
@@ -11,106 +11,143 @@ or modifying files produced by other programs (e.g.,
-*Remembered* along the way).
-At the end, we get a tree sequence that looks something like
-the situation on the right,
-in which many of the nodes and individuals have been removed.
+Suppose we've run a very small simulation with SLiM. The genetic relationships between
+the various diploid individuals who were alive over the course of the simulation might
+look something like the picture on the left below. Note that individuals (circles) are
+diploid, so that each contains two chromosomes or *nodes* (shaded rectangles),
+and that relationships are between the *nodes*, not the individuals.
+At the end of the simulation all we are normally interested in are the genetic
+relationships between the nodes in the *final* generation;
+other parts of the genealogy are irrelevant. To save having to store this unnecessary
+information, SLiM *simplifies* the tree sequence as it goes along, retaining only certain
+parts of the genealogy (primarily, the relationships between the individuals' nodes
+in the current generation). When the tree sequence is output, the result then looks
+something like the situation on the right, in which many of the nodes and individuals
+have been removed.
 
 .. figure:: _static/pedigree01.png
    :scale: 40%
 
+.. _Left in tree sequence:
 
 *************************************
 Who and what is in the tree sequence?
 *************************************
 
-OK, who and what exactly *is* left in the tree sequence after simplification?
-We clearly want all individuals of the *final* generation
-to have their genealogies recorded in the tree sequence.
-in the picture above there are two additional individuals pictured in the
-tree sequence that were not alive at the end of the simulation, because they
-were *remembered*, using the ``treeSeqRememberIndividuals()`` method. The lower
-one (still containing 2 nodes or chromosome) was *permanently* remembered, which is the
-default behaviour. The upper one (with only one node left) was simply marked as *retained*,
-done by passing ``permanent=F`` when calling ``treeSeqRememberIndividuals()``.
-As you might expect, permanently remembered individuals and their nodes are always
-kept in the tree sequence after simplification. In contrast, individuals that are
-merely marked for retention can lose their nodes (as has happened to one of the nodes in
-this case). Furthermore, once all an individuals's nodes are lost, simplification
-removes the individual too, for efficiency reasons.
-
+OK, who and what exactly *is* left in the tree sequence after the unnecessary
+information has been removed? Here's the diagram above with some added terminology. Note
+that for simplicity, in this model the individuals that are currently alive are
+all in the final generation, but more generally, individuals alive at a particular time
+should be identified via the :meth:`.SlimTreeSequence.individuals_alive_at()` method (see
+:ref:`sec_extracting_individuals`).
```

Comment:
this note is getting in to the weeds a bit much - in the diagram it isn't exactly clear who's alive when, or if this is something with overlapping generations or not

Reply:
OK. It was in the previous write-up, but I wasn't quite sure where to slide it in. Note that the diagram isn't right yet - I haven't changed anything there.